docs/docs/about/release-notes.mdx

```diff
@@ -713,10 +714,6 @@ sidebar_position: 4
 - [purview retentionlabel remove](../cmd/purview/retentionlabel/retentionlabel-remove.mdx) - deletes a retention label [#4149](https://github.com/pnp/cli-microsoft365/issues/4149)
 - [purview retentionlabel set](../cmd/purview/retentionlabel/retentionlabel-set.mdx) - updates a retention label [#4148](https://github.com/pnp/cli-microsoft365/issues/4148)
 
-**Search:**
-
-- [search externalconnection schema add](../cmd/search/externalconnection/externalconnection-schema-add.mdx) - adds a schema to a specific external connection [#3171](https://github.com/pnp/cli-microsoft365/issues/3171)
```

Comment:
Instead of removing this, let's just adjust the hyperlink to point to the new docs page.